File: 3rd sem/Java/PrimeNumber.java
import java.util.Scanner;

public class PrimeNumber {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number:");
        int num = sc.nextInt();
        int key = 0;
        for (int i = 2; i < num / 2; i++) {
            if (num % i == 0) {
                key = 1;
            }
        }
        if (key == 1) {
            System.out.println(num + " is not Prime number.");
        } else {
            System.out.println(num + " is Prime number.");
        }

    }
}

File: 4th sem/Python/PrimeNumber.py
num = int(input("Enter a number: "))
flag = 0
if(num == 2):
    print("Prime Number")
else:
    for i in range (2, num):
        if(num % i == 0):
            flag = 1
            break
    if(flag == 1):
        print("Non Prime number")
    else:
        print("Prime Number")



